botrest/src/main/resources/application.yml

```diff
@@ -1,5 +1,5 @@
 spring:
   datasource:
-    url: jdbc:mysql://localhost:3306/BlackBookRobotDB
-    username: root
-    password: 123ZOOMzoom890
```

Comment:
:1st_place_medal: 
My idea is to create file `application.yml.sample` then add to gitignore application.yml, later copy `application.yml.sample` and remove `.sample`.